test/functional/datastoresrp/resources/testdata/datastoresrp-resources-redis-manual.bicep

```diff
@@ -45,33 +45,25 @@ resource redisContainer 'Applications.Core/containers@2023-10-01-preview' = {
       ports: {
         redis: {
           containerPort: 6379
-          provides: redisRoute.id
+          port: 80
         }
       }
     }
     connections: {}
   }
 }
 
-resource redisRoute 'Applications.Core/httproutes@2023-10-01-preview' = {
-  name: 'rds-rte'
-  location: 'global'
-  properties: {
-    application: app.id
-  }
-}
-
 resource redis 'Applications.Datastores/redisCaches@2023-10-01-preview' = {
   name: 'rds-rds'
   location: 'global'
   properties: {
     environment: environment
     application: app.id
     resourceProvisioning: 'manual'
-    host: redisRoute.properties.hostname
-    port: redisRoute.properties.port
+    host: 'rds-ctnr'
+    port: 80
```

Comment:
Instead of hardcoding these, should we get them through the object? Like `redisContainer.name` for example...

Reply:
Applies to all the hardcoded properties...

Reply:
+1

Reply:
I had ask @rynowak  about having hardcoded strings, but he had mentioned its a convention with DNS -SD .. it is viewed not as a hardcoded string, but as a DNS hostname. Please correct me if I should change it.